backend/controllers/github.go

```diff
@@ -486,21 +486,23 @@ func handlePullRequestEvent(gh utils.GithubClientProvider, payload *github.PullR
 	}
 
 	// perform locking/unlocking in backend
-	for _, project := range impactedProjects {
-		prLock := dg_locking.PullRequestLock{
-			InternalLock: locking.BackendDBLock{
-				OrgId: organisationId,
-			},
-			CIService:        ghService,
-			Reporter:         comment_updater.NoopReporter{},
-			ProjectName:      project.Name,
-			ProjectNamespace: repoFullName,
-			PrNumber:         prNumber,
-		}
-		err = PerformLockingActionFromCommand(prLock, *diggerCommand)
-		if err != nil {
-			utils.InitCommentReporter(ghService, prNumber, fmt.Sprintf(":x: Failed perform lock action on project: %v %v", project.Name, err))
-			return fmt.Errorf("failed to perform lock action on project: %v, %v", project.Name, err)
+	if dg_configuration.PrLocks{
```

Comment:
you need to use the config object here:

```suggestion
	if config.PrLocks{
```